### PR TITLE
Upgrade bifrost to 0.2.0 and use fns in interceptors

### PR DESCRIPTION
**Pull request by tie-rack**
The real benefit will come later when we need asynchronous interceptors that don't map directly to a works component's handler.

Pivotal story: [150491144](https://www.pivotaltracker.com/story/show/150491144)

### CODE REVIEW

File: src/turbovote_http_api/handlers/vote_by_mail.clj [hunk 1]
```diff
@@ -2,16 +2,19 @@
   (:require
    [bifrost.core :as bifrost]
    [clojure.core.async :as async]
+   [kehaar.wire-up :as wire-up]
    [turbovote-http-api.handlers.utils :as utils]))
 
 (defonce apply-channel (async/chan 1000))
 
+(def apply-fn (wire-up/async->fn apply-channel))
```
Comment (cap10morgan) [inline src/turbovote_http_api/handlers/vote_by_mail.clj]: Why `apply-fn` here but `create` and `update` elsewhere? Just curious.

Comment (tie-rack) [reply src/turbovote_http_api/handlers/vote_by_mail.clj]: I caught this in my overview before I pushed these changes up. But I couldn't bring myself to make the change to match the others. The name `apply` feels different. Sacrosanct.

I'll make the change if I can add a comment apologizing to McCarthy.

Comment (cap10morgan) [reply src/turbovote_http_api/handlers/vote_by_mail.clj]: Damn, that's such a good argument. I kinda had a similar feeling reading over it and was secretly hoping the reason would be along these lines. You can't just _reject_ `apply` after all.

I am fine with whatever you decide here. But the comment apologizing to McCarthy will make me happy every time I see it. FWTW.

Comment (tie-rack) [reply src/turbovote_http_api/handlers/vote_by_mail.clj]: I can't do it.

**Comment by tie-rack** [comment]
This is worth glancing at again @cap10morgan with the new endpoints covered since the last push (and then we can merge if you like it).

**Pull request merged**